README.md

```diff
@@ -5,12 +5,24 @@ dialog-polyfill.js is a polyfill for `<dialog>`.
 and the
 [HTML spec](http://www.whatwg.org/specs/web-apps/current-work/multipage/commands.html#the-dialog-element).
 
-### Example
+## Usage
+
+### Installation
+
+You may optionally install via Bower-
+
+    $ bower install dialog-polyfill
+
+### Supports
+
+This polyfill works on modern versions of all major browsers. It also supports IE9 and above.
+
+## Example
 
 ```html
 <head>
   <script src="dialog-polyfill.js"></script>
-  <link rel="stylesheet" type="text/css" href="dialog-polyfill.css">
+  <link rel="stylesheet" type="text/css" href="dialog-polyfill.css" />
```

Comment:
As `link` is a [void element](http://www.w3.org/TR/html5/syntax.html#void-elements), it's valid (but optional) for it to have a 'closing' slash. It's my preferred style :)